docs/documentation/helm.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Helm
+description: Helm
+layout: docs
+permalink: /docs/helm
+---
+
+### Helm
+
+Dekorate also supports generating manifests for `helm`. To make use of this feature you need to add:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>helm-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+This module provides the [@HelmChart](annotations/helm-annotations/src/main/java/io/dekorate/helm/annotation/HelmChart.java) that will generate the following Helm resources:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+- <chart name>-<chart version>-helm.tar.gz
+
+#### Getting Started
+
+Add your Helm chart configuration in your properties like:
+
+```
+dekorate.helm.name=myChart
+dekorate.helm.description=Description of my Chart
+```
+
+Or annotate one of your Java source files with the Helm annotation:
+
+```java
+@HelmChart(name = "myChart", description = "Description of my Chart")
+@SpringBootApplication
+public class Main {
+
+  public static void main(String[] args) {
+    SpringApplication.run(Main.class, args);
+  }
+}
+```
+
+#### Building
+
+    mvn clean package
+
+To generate the above Helm resources under the folder `target/classes/META-INF/dekorate/helm/<chart name>/`.
+
+Moreover, assuming you're using Kubernetes, the Helm templates will include the following files by default:
+- deployment.yaml
+- ingress.yaml
+- service.yaml
+- NOTES.txt
+
+#### Mapping Values
+
+By default, Dekorate will generate the Helm values file (`values.yaml`) by mapping the following pre-configured properties:
+
+- The Kubernetes/Openshift replicas
+- The Kubernetes/Openshift image
+- The Kubernetes/Openshift Env Var values (only for plain values - secrets or configmaps are not supported yet)
+- The Kubernetes ingress host
+- The Openshift S2i builder image
+
+For example, if you set 3 replicas for your deployment:
+
+```
+dekorate.helm.name=myChart
+dekorate.helm.description=Description of my Chart
+
+# Set replicas to 3
+dekorate.kubernetes.replicas=3
+```
+
+Dekorate will generate the next Helm values file at `target/classes/META-INF/dekorate/helm/<chart name>/values.yaml`:
+
+```yaml
+---
+myModule:
+  replicas: 3
+```
+
+Note that `myModule` is the name of your project.
```

Comment:
Highlight the Note with something like `**Note**: `

Reply:
Changed